File: DNS/coding-sample/2-pointer/ReverseString.java
import java.util.Arrays;
class ReverseString {
    public static void main(String args[]) {
        String a = "              Hello     World                     ";
        System.out.println(reverseString(a));
    }

    public static String reverseString(String a) {
        // convert String into String array
        String[] array = a.split("\\s+");
        System.out.println(Arrays.toString(array));
        int right = array.length -1;
        int i =0;
        while(i < right) {
            String temp = array[i];
            array[i] = array[right];
            array[right] = temp;
            i++;
            right--;
        }
            return String.join(" ", array);
        }
    }

File: python-interview-question/reverse_string.py
s= "verily"
test = list(s)
print(s[::-1])
index =0
right = len(test) -1
while index <= right:
    print(index, "", right)
    temp = test[index]
    test[index] = test[right]
    test[right] = temp
    index +=1
    right -=1
print("".join(test))